runtime/lua/vim/treesitter/playground.lua

```diff
@@ -270,6 +270,7 @@ function M.inspect_tree(opts)
   vim.bo[b].buftype = 'nofile'
   vim.bo[b].bufhidden = 'wipe'
   vim.bo[b].filetype = 'query'
+  vim.bo[b].query_lint_on = {} -- disable query linter
```

Comment:
This won't work, since `query_lint_on` is a global variable(!)

If we want to be able to disable the linting conditionally -- which we absolutely do, in this case, we need to have an additional buffer variable that can be checked in the autocommand before starting the linter.

Reply:
created a buffer variable with same name. Changing the buffer variable won't change the query events obviously without redoing the ftplugin.

Reply:
Yes, that's fine, we just need a way of skipping the lint per-buffer -- no matter which event.